decoder/PortsPCA9685.h

```diff
@@ -11,6 +11,10 @@
 #include "Ports.h"
 #include "Adafruit_PWMServoDriver.h"
 
+#define MIN_PULSE_WIDTH 650
+#define MAX_PULSE_WIDTH 2350
+#define DEFAULT_PULSE_WIDTH 1500
+#define FREQUENCY 50
```

Comment:
#define in C++ sind böse ;-)

Besser direkt in die Klasse, damit es keine Seiteneffekte zu anderen Code gibt
int maxLog = 50;

Ändere ich nach dem Merge